### PR TITLE
Set Rust 2021 edition.

### PR DESCRIPTION
**Pull request by battlmonstr**
This fixes complication on stable Rust, because vorot93/task-group requires Rust 2021.

### CODE REVIEW

File: rustfmt.toml [hunk 1]
```diff
@@ -1 +1 @@
-imports_granularity = "Crate"
+
```
Comment (vorot93) [inline rustfmt.toml:1]: Stray change?

Comment (battlmonstr) [reply rustfmt.toml:1]: this is a nightly-only feature

Comment (battlmonstr) [reply rustfmt.toml:1]: getting a warning

Comment (battlmonstr) [reply rustfmt.toml:1]: there are no format changes with this

Comment (battlmonstr) [reply rustfmt.toml:1]: this is a nightly-only feature
was getting a warning about this
there are no format changes without this

Comment (vorot93) [reply rustfmt.toml:1]: Looks like it's unstable only in rustfmt.toml. Let's keep it in CI invocation.

**Review by vorot93 — APPROVED** [review]

**Pull request merged**